src/Miso/Internal.hs

```diff
@@ -463,7 +463,7 @@ renderStyles styles =
 -- The 'Sub' can be stopped by calling @stop subName@ from the 'update' function.
 -- All 'Sub' started will be stopped if a 'Component' is unmounted.
 --
-startSub :: SubName -> Sub action -> Effect model action
```

Comment:
>  The 'Sub' can be stopped by calling @stop subName@ from the 'update' function

Could you please also update mentions of subName in haddocks?

Reply:
`subName` can now mean `Ord subName => subName`, so it's technically still valid

Reply:
used `subKey`